persistence-api/pom.xml

```diff
@@ -15,6 +15,21 @@
       <url>https://repo.maven.apache.org/maven2</url>
     </repository>
   </repositories>
+  <dependencyManagement>
+    <dependencies>
+      <!-- 30-Mar-2022, tatu: Need to force Netty overrides here so
```

Comment:
<Narrator>: this does not -- alas! -- force upgrade from baseline of `java-driver-core`. But it is enough to ensure baseline is at least Netty 4.1.60 (from java-driver-core) which covers all current CVEs. Which is our immediate goal.